CLC docs updates (#318)
hazelcast/management-center-docs: docs/modules/tools/pages/console.adoc

WARNING: The console is not recommended for production use.

> TIP: Embedded CLC only supports the PKCS12 store type for TLS. If you're using any other TLS configuration, you'll need to convert it to PKCS12 to use it with CLC.

Reviewer: ```suggestion
TIP: Embedded CLC only supports the PKCS12 store type for TLS. If you're using any other TLS configuration, you must convert it to PKCS12 before using it with CLC.
```

TIP: The console container is resizable; to change the height, drag the bar on the top of the console.
You can maximize or minimize the console container using the icons in its top-right corner.

> WARNING: CLC binary is not signed for Windows users.

Reviewer: ```suggestion
WARNING: The CLC binary is not signed for Windows users.
```

Reviewer: Should the next two lines should be part of the warning, or are they more general (that is, not only considerations for Windows users)?

> You may need to grant access to it or sign it manually in order to use it.

Reviewer: ```suggestion
To use the binary, you might have to grant it access or manually sign it.
```

You can find the binary inside the clc folder in the Management Center home directory.

> If you're encountering difficulties starting the console, it's recommended to check the access permissions for the `clc` folder and the CLC binary.

Reviewer: ```suggestion
If you're encountering difficulties starting the console, we recommend that you check the access permissions for the `clc` folder and the CLC binary.
```

> The `clc` folder is created under the Management Center home directory, and all console-related files are stored there, including the binary.

Reviewer: ```suggestion
The `clc` folder is created under the Management Center home directory. All console-related files are stored in this directory, including the binary.
```

> You can navigate to the Management Center home directory and try running the CLC binary manually to see if everything is working correctly.

Reviewer: ```suggestion
You can navigate to the Management Center home directory and run the CLC binary manually to see if everything is working correctly.
```
